node/src/Pos/Wallet/Web/State/State.hs

```diff
@@ -181,7 +181,7 @@ getUpdates = queryDisk A.GetUpdates
 getNextUpdate :: WebWalletModeDB ctx m => m (Maybe CUpdateInfo)
 getNextUpdate = queryDisk A.GetNextUpdate
 
-getHistoryCache :: WebWalletModeDB ctx m => CId Wal -> m (Maybe (Map TxId TxHistoryEntry))
+getHistoryCache :: WebWalletModeDB ctx m => CId Wal -> m (Map TxId TxHistoryEntry)
```

Comment:
Oh, this looks outside the changes of simply logging. What was this necessary?

Reply:
Strangely but it somehow relates.
Previously as part of one fix all history-related functions to access / modify database were rewritten in a way that, if wallet doesn't exist, empty history for it is returned.
But this change was partly lost during rebase, and now we have very annoying but inappropriate warning here: https://github.com/input-output-hk/cardano-sl/pull/1858/files#diff-1bfb54b702d6d93310adefeed510d053L52
(@fersel complained on it twice already)

I can create separate PR if needed where this change will be adopted